src/app/routes.tsx

```diff
@@ -1,12 +1,13 @@
-import { About, Description, Home, NotFound } from './components';
+import { About, Description, NotFound } from '@/components';
+import { Home } from '@/pages/home/home';
 
 export const routes = [
   {
     path: '/',
     element: <Home />,
     children: [
       {
```

Comment:
При такой настройке маршрутов никогда не отображается страница 404.  
Для картинки ниже ожидаемое мной поведение - отображение страницы not found:
<img width="1019" height="883" alt="image" src="https://github.com/user-attachments/assets/829d496f-1e5c-491a-9c47-c7602a9fb06a" />

возможно, стоит поправить так:
```suggestion
    path: '/pokemons',
    element: <Home />,
    children: [
      {
```